data-tool/scripts/colin_corps_extract_postgres_views_ddl

```diff
@@ -660,7 +660,11 @@ FROM (
                                   'FILE_NOCDR', 'FILE_AM_DI', -- CoD
                                   'FILE_NOALA', 'FILE_NOALB', 'FILE_NOALU', 'FILE_NOALC', 'FILE_AM_BC', 'FILE_AM_LI', 'FILE_AM_RM', 'FILE_AM_SS', -- Alteration
                                   'FILE_COGS1',
-                                  'ADCORP_NULL'
+                                  'ADCORP_NULL',
+                                  'FILE_CHGJU', 'FILE_NWPTA', 'FILE_PARES', 'FILE_TILAT', 'FILE_TILHO', 'FILE_TILMA',
+                                  'SYST_CANPS', 'SYST_CHGJU', 'SYST_CHGPN', 'SYST_CO_PN', 'SYST_LNKPS', 'SYST_NWPTA',
+                                  'SYST_PARES', 'SYST_RIPFL', 'SYST_TILAT', 'SYST_TILHO', 'SYST_NULL',
+                                  'TRESP_NULL', 'TRESP_COUTI'
```

Comment:
can we group and comment indicating that these are skipped?  I think maybe FILE_COGS1 was skipped too?